web/templates/pages/task_edit.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+
+{% block custom_css %}
+	<link rel="stylesheet" type="text/css" href="{% static 'css/jquery.datetimepicker.css' %}" >
+{% endblock %}
+
+{% block content %}
+
+	{% if invalid_project_id %}
+		<p class="alert alert-warning">This task does not exists in your projects list.</p>
+		<p>Return to <a href="{% url 'pages-tasks' %}">Tasks</a></p>
+	{%  else %}	
+		<h1>{% if editing %}Edit{% else %}Add{% endif %} your task</h1>
+		<br />
+		<div>
+			<form method="POST">{% csrf_token %}
+			{{ taskform.as_p }}
+			<button type="submit" class="save btn btn-default">Save</button>
+			</form>
+		</div>
+	{% endif %}	
+
+{% endblock content %}
+
+{% block custom_js %}
+	<script type="text/javascript" src="{% static 'js/jquery.datetimepicker.js' %}"></script>
```

Comment:
The datetimepicker script is now enclosed in the base.html template, so you don't need the entire custom_js block